manifests/compose.pp

```diff
@@ -89,10 +89,11 @@
         ensure_packages(['curl'])
       }
 
+      $compose_install = "curl -s -S -L ${proxy_opt} ${docker_compose_url} -o ${docker_compose_location_versioned}"
```

Comment:
Should this not be broken in to args?

```suggestion
      $compose_install = ['curl', '-s', '-S', '-L', $proxy_opt, docker_compose_url, '-o', $docker_compose_location_versioned]"
```

Reply:
Anything that is not broken into args will cry, scream and kick at me when I run test cases. Tried to figure out the reason behind those failures but eventually had to give up and just let some commands stay the same.